dotCMS/src/main/java/com/dotcms/rendering/velocity/directive/DotCacheDirective.java

```diff
@@ -59,8 +59,7 @@ public boolean render(InternalContextAdapter context, Writer writer, Node node)
             return true;
         }
 
-        final StringWriter blockContent = STRINGWRITER.get();
-        blockContent.getBuffer().setLength(0);
+        final StringWriter blockContent = new StringWriter();
```

Comment:
so are we not going to use the 'STRINGWRITER.get();' anymore?, if it is the why are we not removing the object

https://github.com/dotCMS/core/blob/1085a2ded4d8cbe91423093eb92de16b493eb9ae/dotCMS/src/main/java/com/dotcms/rendering/velocity/directive/DotCacheDirective.java#L77

I think we are not using it in any other place

Reply:
the **STRINGWRITER.get()** is being removed https://github.com/dotCMS/core/pull/24274/files#diff-c1899c2fed882635c567d96a1d88783ad96bc6fb6c05e7fe3eb6415082ee3335L77-L78